scripts/india_lgd/local_government_directory_districts/preprocess.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import csv
+import difflib
+import pandas as pd
+
+manual_override = {
+    "delhi": {
+        "south east": "south east delhi"
+    },
+    "assam": {
+        "karbi anglong": "east karbi anglong"
+    },
+    "chhattisgarh": {
+        "gaurella pendra marwahi": "gaurela-pendra-marwahi"
+    },
+    "gujarat": {
+        "chhotaudepur": "chhota udaipur",
+        "devbhumi dwarka": "devbhoomi dwarka"
+    },
+    "telangana": {
+        "jagitial": "jagtial",
+        "jangoan": "jangaon"
+    },
+}
+
+
+class LocalGovermentDirectoryDistrictsDataLoader:
+
+    def __init__(self, lgd_csv, wikidata_csv, clean_csv):
+        self.lgd_csv = lgd_csv
+        self.wikidata_csv = wikidata_csv
+        self.clean_csv = clean_csv
+        self.lgd_df = None
+        self.wikidata_df = None
+        self.clean_df = None
+
+    @staticmethod
+    def format_title(s):
+        # Converts to title case, except for the words like `of`, `and` etc
+        name_list = s.split(' ')
+        first_list = [name_list[0].capitalize()]
+        for name in name_list[1:]:
+            first_list.append(name if name in
+                              ["of", "and"] else name.capitalize())
```

Comment:
just for safety should we have spaces around of and and?

Reply:
I do split based on space. That should handle it no?

`name_list = s.split(' ')`

Reply:
Ah, makes sense!